build/buildBuildImages.sh

```diff
@@ -36,28 +36,32 @@ labelContent="git_commit=$GIT_COMMIT, build_number=$BUILD_NUMBER, release_tag_na
 # https://medium.com/@Drew_Stokes/bash-argument-parsing-54f3b81a6a8f
 PARAMS=""
 while (( "$#" )); do
-	case "$1" in
-		-t|--type)
-		imageTypeToBuild=$2
-		shift 2
-		;;
-		-s|--sdk-storage-account-url)
-		sdkStorageAccountUrl=$2
-		shift 2
-		;;
-		--) # end argument parsing
-		shift
-		break
-		;;
-		-*|--*=) # unsupported flags
-		echo "Error: Unsupported flag $1" >&2
-		exit 1
-		;;
-		*) # preserve positional arguments
-		PARAMS="$PARAMS $1"
-		shift
-		;;
-	esac
+  case "$1" in
+    -token|--sas-token)
```

Comment:
- [ ] As discussed in the meeting yesterday, lets remove this and implement the logic to detect if being run locally, and if so running the script to load the token into a local variable w/o printing it